.github/workflows/tests.yml

```
@@ -45,21 +45,14 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: ${{ github.event.pull_request.head.sha }}
-          path: jfrog-cli
 
-      - name: Checkout testing infra
-        uses: actions/checkout@v2
-        with:
-          repository: RobiNino/jfrog-testing-infra #todo change to jfrog
-          path: jfrog-testing-infra
-          ref: rt-setup #todo remove
-
-      - name: Setup Artifactory
+      - name: Setup Artifactory # TODO change to jfrog
         run: |
-          cd jfrog-testing-infra
-          go run main.go
+          go install github.com/RobiNino/jfrog-testing-infra/local-rt-setup@latest
```

I think the latest is the default
```suggestion
          go install github.com/RobiNino/jfrog-testing-infra/local-rt-setup
```

---

Tag is required:
```
$ go install github.com/RobiNino/jfrog-testing-infra/go
no required module provides package github.com/RobiNino/jfrog-testing-infra/go; to add it:
	go get github.com/RobiNino/jfrog-testing-infra/go